### ISSUE TITLE
Searching for a concept delivers strange results

### ISSUE THREAD

**Issue by dpancic** [SSHOC/sshoc-marketplace-backend#51]
In GitLab by @KlausIllmayer on Jan 25, 2021, 14:20

I'm trying to find a specific language in the iso-639-3-v2 vocabulary. I use the API call to concept-search and look for the multiple-language tag (mul): https://marketplace-api.sshoc.eu/api/concept-search?q=mul&types=language&perpage=100
I'm getting back 54 hits where for most of them I'm unclear, why they are returned. Some like the emm-language ("Mamulique") do have "mul" in the label but for many I don't see, why they are returned. And even worse, I don't find the language with the code "mul". It seems that it doesn't search in the code-field but returns languages where I don't see any clue, why they are returned. What do I miss here?

"mul" is there, you can find it as entry 97 in this call: https://marketplace-api.sshoc.eu/api/vocabularies/iso-639-3-v2?perpage=100&page=42

Okay, looking here it seems that q is case-sensitive, calling https://marketplace-api.sshoc.eu/api/concept-search?q=Mul&types=language&perpage=100&advanced=false I find the label "Multiple languages". But there are also a lot of items, where I'm not sure, why they show up - and it seems to confirm, that q does not search in the field code.

Summing up:
* Why are there so many return values where I can't find a clue, why it is returned (is this something Solr specific)?
* Can we have a tag to switch to case-insensitive search?
* Can we search also in code? For concept-search I guess it would make sense.
